src/App.js

```diff
@@ -4,15 +4,67 @@ import SignUpPage from "./Components/Pages/SignUpPage";
 import LoginPage from "./Components/Pages/LoginPage";
 import ItemDetailsPage from "./Components/Pages/ItemDetailsPage";
 
+const dummyData = [
```

Comment:
From my previous internship experience, it is **recommended** to have the data stored in a _separate data file_ in XML, CSV, or any other format 🙂

I suppose that the data on the main file is a temporary solution?

Reply:
Yes, this is only to showcase what the front-end looks like with data.

We will implement back-end for next sprint.